xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
@FernetMenta any idea why this happens ? If I place back the second (bool) offscreen argument again  it does compile just fine. When I leave it out, the compiler complains about a missing default constructor. Adding an empty default constructor which takes no parameters makes the code compile again but Kodi will freeze on execution as the thread guard is bypassed. The solution as reached out by @Rechi with explicitly defining the classname does the trick but I don't understand why. If it's no big deal, I can change it and this PR is done, if it's unsure maybe better to close the PR and wait for v18 ? It's not that big of a deal. Users face an incidental lockup of Kodi caused by Python adding which will be fixed in v18.